### ISSUE TITLE
CLP Mural Design  - 508/a11y Review 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#18756]
## Description: 
_Looking for a review of the below Mural design created in support of Public Websites Campaign Landing Page.  The design has been reviewed previously via Jennifer Strickland however a gap of time as occurred and would like a secondary review to ensure the design is 508/a11y complaint as we build out FE component._

[Mural Design](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1607363793772/b7f9e809d68ace9c8d75ab649a758ed7653c8461)

Please Thread here for any feedback for Public Websites Team.  

    - [508-defect-0 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0)
    - [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)
    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)
    - [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)
    - [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

**Comment by joshkimux** [comment]
If this is a link `a`, it should be styled as a link instead of a button. [You can read more on this recommendation on this slack thread between several accessibility specialists](https://dsva.slack.com/archives/C8E985R32/p1610659694009400?thread_ts=1610652897.002600&cid=C8E985R32), but I think it's best described by the last comment in that thread:

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to."

<img width="951" alt="Screen Shot 2021-01-21 at 12 19 41 PM" src="https://user-images.githubusercontent.com/14154792/105386879-f1a83000-5be2-11eb-9047-deb50fda6505.png">

**Comment by joshkimux** [comment]
1. Was there a reason for designing this page with an image? 
2. If so, do we anticipate it ever being used to convey information by content creators?

If there's a possibility `field_image` will be used to convey information (which I'm assuming since `field_img:alt` is listed in the mural), we'll need to think about how to write it up and enforce rules on alt text.

- Given the way it is styled, this may be written as a CSS background image which _cannot_ carry an `alt` attribute. [This article lists a couple of ways of approaching this problem.](https://www.davidmacd.com/blog/alternate-text-for-css-background-images.html)
- It may also be worth considering not providing the opportunity to have a background image at all to prevent content creators from using it to convey information (e.g. as a banner to give details about an event).

<img width="781" alt="Screen Shot 2021-01-21 at 12 39 57 PM" src="https://user-images.githubusercontent.com/14154792/105389390-c83cd380-5be5-11eb-992f-195a4e404ed9.png">

**Comment by joshkimux** [comment]
Would it be possible to include transcripts? They meet a wide range of user needs e.g. for people who are both Deaf and blind, people with low bandwidth connections, etc. [More guidance on transcripts from W3C](https://www.w3.org/WAI/media/av/transcripts/).

<img width="1223" alt="Screen Shot 2021-01-21 at 12 53 32 PM" src="https://user-images.githubusercontent.com/14154792/105391318-01764300-5be8-11eb-86ca-834e1dba151c.png">

**Comment by joshkimux** [comment]
<img width="1180" alt="Screen Shot 2021-01-21 at 2 06 30 PM" src="https://user-images.githubusercontent.com/14154792/105399681-f6281500-5bf1-11eb-9b2a-28843cded651.png">

Will instructions for how to write alt text along with a form field be provided?

**Comment by joshkimux** [comment]
The grey text used for some strings on the page may fail to meet contrast requirements.

<img width="774" alt="Screen Shot 2021-01-21 at 2 11 15 PM" src="https://user-images.githubusercontent.com/14154792/105400213-a1d16500-5bf2-11eb-81b3-1ed8f020059b.png">
<img width="270" alt="Screen Shot 2021-01-21 at 2 09 56 PM" src="https://user-images.githubusercontent.com/14154792/105400214-a269fb80-5bf2-11eb-8b73-2d79c9b47555.png">

**Comment by joshkimux** [comment]
Consider reducing the verbosity of social media links by attaching an `aria-describedby` that references the heading to this `ul` of contact links.

<img width="950" alt="Screen Shot 2021-01-21 at 2 22 41 PM" src="https://user-images.githubusercontent.com/14154792/105402303-4359b600-5bf5-11eb-99ef-aee12b66a148.png">

**Comment by joshkimux** [comment]
@brianalloyd I've finished my a11y review! Overall, I thought these wires were thoughtfully put together 💯  
Do let me know if you or the team have any questions about the recommendations and I'd be more than happy to explain in more detail! 🙇

**Comment by brianalloyd** [comment]
@kevwalsh in review I believe these would be helpful considerations from a CMS / Design standpoint.  Perhaps we can discuss during our next sync or before.  CC @johnhashva

**Comment by kevwalsh** [comment]
@brianalloyd 

Issues that have some CMS relevance

**1. Video transcripts**

CMS can certainly add a transcript field if that's a priority, but then we're adding functionality that Youtube doesn't necessarily support upstream.

**2. Alt text fields** 

CMS editors are required to provide alt text. Here's what it looks like (not covered in the mural screenshots)

![Edit_Image_Tauna_Perenovich_20200901_0018_jpg___VA_gov_CMS](https://user-images.githubusercontent.com/643678/105411174-0a274300-5c01-11eb-8b7d-e17ad51c6736.png)

**3. Connect with us verbosity** (eg Veterans Health Administration being repeated)

Not really any role for CMS top play in resolving this, but worth noting that the same pattern is on Benefits Hub Pages like https://www.va.gov/health-care/ , and the same content will power both products.

**Comment by johnhashva** [comment]
@kevwalsh thanks for quick response/updates.   Re the Video panel:

- **Video panel defect-level-2, potentially 1 if captions are unavailable** Since this is an embed (or iframe) of the YT video and the YT player includes close captioning functionality -- do we inherit that as part of the player?

**Comment by joshkimux** [comment]
@kevwalsh Thank you for the additional context and review! Some additional notes:

1. It would amazing if transcripts could be added as a field even if Youtube doesn't necessarily support it upstream. It provides a great, and often more comfortable, alternative to having to watch an actual video for just about everyone-- especially for people with low bandwidths or who are both Deaf and blind! There's also SEO benefits and it helps users search /scan through information faster when they're short on time 😄 

2. Would it be possible for instructions to be included on how to write alt text for the CMS editors? e.g. [attaching a link to WebAIM's guidance?](https://webaim.org/techniques/alttext/)

**Comment by kevwalsh** [comment]
@johnhashva first, i had a quick chat with @kelsonic about CC today, and Youtube supports closed captioning out of the box in the iframes. the text can be specified manually by the video owner in youtube, otherwise is generated automatically by speech recognition. 

I agree with @joshkimux  transcripts can be great for all users and for accessibility SEO.  A bit outside my lane here, but it seems like it might be worth waiting until we've had a chance to think more holistically about Digital Asset Management and the future of VA Youtube integration with the CMS before adding functionality to the CMS that doesn't exist in youtube. Who would create these transcripts? Do media production teams at VA already do this, as part of the manual CC? lots to figure out there. 

@joshkimux On alt text guidance, we would very much welcome collaboration with you on how best to write it. We actually have an issue about it on our backlog, https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3444 , based on a conversation with @jenstrickland a couple months ago.  I'll bump that issue forward and find a time to meet about that, with our UX writer, you and Jen. Thanks for bumping this!

**Comment by johnhashva** [comment]
@kevwalsh thanks!  Good to know that CC will be in place.  As for transcripts, @brianalloyd and I will give this more thought and put in the backlog for CLP -- as you note, part of the larger CMS DAM (digital asset management) consideration .   Other words: Not an MVP priority -- but important for future state.

**Comment by joshkimux** [comment]
@johnhashva For some added context on the importance of transcripts. 

"Transcripts are the only way a person who is deafblind will be able to access all of the information communicated in audio and video formats. Some modern braille readers have the ability to display video captions, but it can be challenging to read the braille characters as quickly as the captions are displayed, so transcripts are still the best option for deafblind users." - Deque

**Comment by brianalloyd** [comment]
Completed from a Public Websites standpoint.  Have created a ticket for the CC transcript for backlog grooming for CLP forward.  Closing this ticket in support.
